src/main/java/seedu/address/logic/parser/MarkTutorialCommandParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.MarkTutorialCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+
+
+
+/**
+ * Parses input argument and creates a new {@code MarkTutorialCommand} object.
+ */
+public class MarkTutorialCommandParser implements Parser<MarkTutorialCommand> {
+
+    @Override
+    public MarkTutorialCommand parse(String args) throws ParseException {
+
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(args);
```

Comment:
Maybe you can declare `Index index` in the try block to make the code shorter?

Reply:
Hmm, I don't think it would make a difference, not sure if I should declare in a try block because it is a variable.

Reply:
Added in as I think it'll make the code neater

Reply:
I pointed it out because this was how it was implemented in the `DeleteCommandParser` (not that it's the absolute correct implementation)